example/README.md

```diff
@@ -29,7 +30,8 @@ kind load docker-image --name topolvm-example ghcr.io/topolvm/pvc-autoresizer:de
 Deploy `pvc-autoresizer`:
 
 ```
-make deploy
+helm repo add pvc-autoresizer https://topolvm.github.io/pvc-autoresizer/
```

Comment:
It would be better to have `helm repo update` after `helm repo add`.

Reply:
updated